server/src/models/House.ts

```diff
@@ -9,11 +11,13 @@ export class House implements MapPositionable {
 
     hp: number = 500
     team: Team = 'Human'
+    minionSpawner: MinionSpawner
 
-    constructor(team: Team, mapHouse: MapHouse) {
+    constructor(team: Team, mapHouse: MapHouse, minionSpawnPoint: MapSpawnPoint) {
         this.team = team
         this.id = mapHouse.id
         this.position.x = mapHouse.position.x
         this.position.y = mapHouse.position.y
+        this.minionSpawner = new MinionSpawner(team, minionSpawnPoint)
     }
```

Comment:
After merging my PR, use it as an example of how to spawn minions only during the right time of day for your team, as I do with food.

Reply:
Oh yea, that is totally missing right now